.github/workflows/gerrit-ci-management-merge.yaml

```diff
@@ -0,0 +1,87 @@
+---
+name: Gerrit ci-management Merge
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+    inputs:
+      GERRIT_BRANCH:
+        description: "Branch that change is against"
+        required: true
+        type: string
+      GERRIT_CHANGE_ID:
+        description: "The ID for the change"
+        required: true
+        type: string
+      GERRIT_CHANGE_NUMBER:
+        description: "The Gerrit number"
+        required: true
+        type: string
+      GERRIT_CHANGE_URL:
+        description: "URL to the change"
+        required: true
+        type: string
+      GERRIT_EVENT_TYPE:
+        description: "Type of Gerrit event"
+        required: true
+        type: string
+      GERRIT_PATCHSET_NUMBER:
+        description: "The patch number for the change"
+        required: true
+        type: string
+      GERRIT_PATCHSET_REVISION:
+        description: "The revision sha"
+        required: true
+        type: string
+      GERRIT_PROJECT:
+        description: "Project in Gerrit"
+        required: true
+        type: string
+      GERRIT_REFSPEC:
+        description: "Gerrit refspec of change"
+        required: true
+        type: string
+    secrets:
+      GERRIT_SSH_PRIVKEY:
+        description: "SSH Key for the authorized user account"
+        required: true
+
+concurrency:
+  group: ${{ github.event.inputs.GERRIT_CHANGE_ID || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  jjb-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lfit/checkout-gerrit-change-action@v0.3
+        with:
+          gerrit-refspec: ${{ inputs.GERRIT_REFSPEC }}
+          delay: "0s"
+      - uses: actions/setup-python@v4
+        id: setup-python
+        with:
+          python-version: "3.11"
+      - name: Clone git submodules
+        run: git submodule update --init
+      - name: Run JJB Merge
+        env:
+          JJB_WORKERS: ${{ vars.JJB_WORKERS }}
+        run: |
+          python -m pip install --upgrade pip
+          pip install jenkins-job-builder
+          mkdir -p "${HOME}/.config/jenkins_jobs"
+          cat << EOF > "${HOME}/.config/jenkins_jobs/jenkins_jobs.ini"
+          [job_builder]
+          ignore_cache=True
+          keep_descriptions=False
+          include_path=.
+          recursive=True
+          query_plugins_info=False
+          config-xml=True
+          EOF
+          workers="${JJB_WORKERS:-0}"
+          jenkins-jobs update --recursive --delete-old --workers "$workers" jjb/
+      - name: clean up temp files
+        shell: bash
+        run: rm -f ${{ github.run_id }}.properties
```

Comment:
To be on the safe side you should also be deleting the ~/.config/jenkins_jobs/jenkins_jobs.ini file at the end. This step should run with an `if: always` definition

Reply:
done